packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -37,13 +38,24 @@ export default class QuanticNumericFacet extends LightningElement {
   )} - ${new Intl.NumberFormat(LOCALE).format(
     item.end
   )}`;
-
+  /** @type {boolean} */
+  @api noInput;
```

Comment:
Is that intended to enable the custom filter by default? It seems like a feature that should be enabled when needed. Unless I am mistaken, [it is working that way in atomic](https://github.com/coveo/ui-kit/blob/master/packages/atomic/src/components/facets/atomic-numeric-facet/atomic-numeric-facet.tsx#L121).

Reply:
Ah yes you're right the default doesn't have it.